### PR TITLE
CL 15 December 2022 release

### PR DESCRIPTION
**Pull request by bvarner-ebi**
CL 15 December 2022 release

### CODE REVIEW

**Comment by bvarner-ebi** [comment]
Removing dbxref annotation on 'valve interstitial cell' (inadvertently added during term creation).

**Comment by bvarner-ebi** [comment]
Cursory review: expected new terms and edits have been verified.

**Review by shawntanzk — APPROVED** [review]
All looks good
Would be good to revert the changes in docs.yml -> however @anitacaron suggests we remove it altogether. 
Something to double check -> the removal of `subset: human_reference_atlas` are intentional. They do look like they are.

File: cl-base.obo [hunk 1]
```diff
@@ -23306,11 +24477,9 @@ relationship: BFO:0000050 UBERON:0002367
 id: CL:1000301
 name: fibroblast of subepithelial connective tissue of prostatic gland
 def: "A fibroblast that is part of the subepithelial connective tissue of prostatic gland." [GOC:tfm]
-subset: human_reference_atlas
```
Comment (bvarner-ebi) [inline cl-base.obo:23309]: @anitacaron, I believe you commented on edits to the human_reference_atlas before. I don't interact with this subset... do you know where these changes originate?

Comment (anitacaron) [reply cl-base.obo:23309]: Yes, these changes are linked to the changes on the ASCT+B tables. No need to worry about them.

**Comment by anitacaron** [comment]
@bvarner-ebi Please, remove the change in docs because the diff here was a manual change, which should not have happened because this file comes from ODK. 😄

**Comment by anitacaron** [comment]
Sorry, it's correct this way.

**Review by anitacaron — APPROVED** [review]

**Pull request merged**